.drone.yml

```diff
@@ -5714,168 +5182,6017 @@ volumes:
+    -d'|' -f 2 | tr -d ' ' | head -n 1)
+  - echo "Found APT package, downloading \"${PACKAGE_NAME}=${FULL_VERSION}\"..."
+  - mkdir -pv "/go/artifacts/deb/$(cat '/go/vars/full-version/v10')"
+  - cd "/go/artifacts/deb/$(cat '/go/vars/full-version/v10')"
+  - apt download ${PACKAGE_NAME}=$FULL_VERSION
+  - mv *.deb "/go/artifacts/deb/$(cat '/go/vars/full-version/v10')/teleport.deb"
+  - curl -Ls -o "/go/build/Dockerfile" "https://raw.githubusercontent.com/gravitational/teleport/${DRONE_SOURCE_BRANCH:-master}/build.assets/charts/Dockerfile"
+  depends_on:
+  - Find the latest available semver for v10
+  - Wait for docker
+  - Check out code
+- name: Build teleport image "teleport-v10-amd64"
   image: docker
   commands:
-  - apk add --no-cache aws-cli
-  - export VERSION=${DRONE_TAG##v}
-  - aws ecr get-login-password --region=us-west-2 | docker login -u="AWS" --password-stdin
-    146628656107.dkr.ecr.us-west-2.amazonaws.com
-  - echo "---> Pulling images for $${VERSION}"
-  - docker pull 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport:$${VERSION}
-  - docker pull 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-ent:$${VERSION}
-  - docker pull 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-ent:$${VERSION}-fips
-  - docker pull 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-operator:$${VERSION}
-  - echo "---> Tagging images for $${VERSION}"
-  - docker tag 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport:$${VERSION}
-    public.ecr.aws/gravitational/teleport:$${VERSION}
-  - docker tag 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-ent:$${VERSION}
-    public.ecr.aws/gravitational/teleport-ent:$${VERSION}
-  - docker tag 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-ent:$${VERSION}-fips
-    public.ecr.aws/gravitational/teleport-ent:$${VERSION}-fips
-  - docker tag 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-operator:$${VERSION}
-    public.ecr.aws/gravitational/teleport-operator:$${VERSION}
-  - docker logout 146628656107.dkr.ecr.us-west-2.amazonaws.com
-  - aws ecr-public get-login-password --region=us-east-1 | docker login -u="AWS" --password-stdin
-    public.ecr.aws
-  - echo "---> Pushing images for $${VERSION}"
-  - docker push public.ecr.aws/gravitational/teleport:$${VERSION}
-  - docker push public.ecr.aws/gravitational/teleport-ent:$${VERSION}
-  - docker push public.ecr.aws/gravitational/teleport-ent:$${VERSION}-fips
-  - docker push public.ecr.aws/gravitational/teleport-operator:$${VERSION}
-  environment:
-    AWS_ACCESS_KEY_ID:
-      from_secret: PRODUCTION_TELEPORT_DRONE_USER_ECR_KEY
-    AWS_SECRET_ACCESS_KEY:
-      from_secret: PRODUCTION_TELEPORT_DRONE_USER_ECR_SECRET
+  - mkdir -p "/go/build" && cd "/go/build"
+  - docker build --target "teleport" --platform "linux/amd64" --tag "teleport-v10-amd64"
```

Comment:
`--tag "teleport-v10-amd64"`

Wouldn't this mean that users would have to pull the image as `teleport:teleport-v10-amd64`? We should use just version as a tag as before.